src/main/kotlin/com/criptext/mail/scenes/mailbox/feed/ui/FeedScene.kt

```diff
@@ -32,13 +34,18 @@ interface FeedScene {
                    feedEventListener: FeedItemHolder.FeedEventListener)
     fun updateFeedBadge(totalNewFeeds: Int)
     fun initializeMenu(menu: IHostActivity.IActivityMenu)
+    fun showStartGuideNotification()
 
     class Default(container: View): FeedScene {
 
         private val recyclerViewFeed: RecyclerView by lazy {
             container.findViewById<RecyclerView>(R.id.recyclerViewFeed)
         }
 
+        private val context = container.context
+
+        private val showStartGuideNotification = StartGuideTapped(context)
```

Comment:
You can initialize this in the function `showStartGuideNotification` cause you only need it once. Make `showStartGuideNotification` nullable